provider/local/converge.go

```diff
@@ -304,6 +306,8 @@ func (p *Provider) resourceURL(app, kind, name string) (string, error) {
 		return fmt.Sprintf("postgres://postgres:password@%s.resource.%s.%s:5432/app?sslmode=disable", name, app, p.Rack), nil
 	case "redis":
 		return fmt.Sprintf("redis://%s.resource.%s.%s:6379/0", name, app, p.Rack), nil
+	case "memcached":
+		return fmt.Sprintf("%s.resource.%s.%s:11211", name, app, p.Rack), nil
```

Comment:
This should probably start with memcached:// for consistency with the other resources.